README.md

```diff
@@ -27,7 +27,7 @@
   - [How to install](#how-to-install)
 * [Getting Started](#getting-started)
 * [webpack CLI Scaffolds](#webpack-cli-scaffolds)
-* Commands
+* [Commands](#commands)
   - [`webpack-cli init`](./packages/init/README.md#webpack-cli-init)
```

Comment:
Should we still keep all list of commands here? 🤔

Reply:
I like the new one, I would remove this 👍

Reply:
Removing, let's see how everyone else finds it :)